pyignite/aio_client.py

```diff
@@ -487,3 +489,21 @@ def get_cluster(self) -> 'AioCluster':
         :return: :py:class:`~pyignite.aio_cluster.AioCluster` instance.
         """
         return AioCluster(self)
+
+    def tx_start(self, concurrency: TransactionConcurrency = TransactionConcurrency.PESSIMISTIC,
+                 isolation: TransactionIsolation = TransactionIsolation.REPEATABLE_READ,
+                 timeout: Union[int, float] = 0, label: Optional[str] = None) -> 'AioTransaction':
+        """
+        Start async thin client transaction.
+
+        :param concurrency: (optional) transaction concurrency, see
+                :py:class:`~pyignite.datatypes.transactions.TransactionConcurrency`
+        :param isolation: (optional) transaction isolation level, see
+                :py:class:`~pyignite.datatypes.transactions.TransactionIsolation`
+        :param timeout: (optional) transaction timeout in seconds if float, in millis if int
+        :param label: (optional) transaction label.
+        :return: :py:class:`~pyignite.transaction.AioTransaction` instance.
+        """
+        if sys.version_info < (3, 7):
+            raise NotSupportedError(f"Transactions are not supported in async client on current python {sys.version}")
```

Comment:
Why python 3.7 is not supported? Does it has to do something with `contextvars`?

Reply:
Python 3.7 is supported. Python 3.6 is not supported, because there is no good backport of contextvars for async stuff to python 3.6.